### PR TITLE
ClayModule

### PR DESCRIPTION
**Pull request by namebee**

### CODE REVIEW

**Comment by dzwdz** [comment]
did you check the amount of clay generated with an xray mod?

File: src/main/java/io/github/protonmc/proton/mixin/world/DefaultBiomeFeaturesMixin.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import net.minecraft.world.gen.feature.DefaultBiomeFeatures;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(DefaultBiomeFeatures.class)
+public class DefaultBiomeFeaturesMixin {
+
+    @Inject(at = @At("HEAD"), method = "addMineables")
```
Comment (YTG1234) [inline src/main/java/io/github/protonmc/proton/mixin/world/DefaultBiomeFeaturesMixin.java:16]: I always prefer injections like this being at `RETURN`

**Review by YTG1234 — CHANGES_REQUESTED** [review]

**Comment by namebee** [comment]
Yeah, With the /fill command. The clay amount is in my opinion fine. @dzwdz

File: src/main/java/io/github/protonmc/proton/mixin/world/DefaultBiomeFeaturesMixin.java (hunk 1 above)
Comment (namebee) [reply src/main/java/io/github/protonmc/proton/mixin/world/DefaultBiomeFeaturesMixin.java:16]: I do not agree with this, As some other mod could potentially mess stuff up.

**Review by YTG1234 — APPROVED** [review]

**Pull request merged**